simpleocr/opencv_utils.py

```diff
@@ -127,8 +128,8 @@ def get_opencv_version():
     Return the OpenCV version by checking cv2.__version__
     :return: int, 2 or 3
     """
-    if cv2.__version__.startswith("3"):
-        return 3
+    if version.parse(cv2.__version__) >= version.parse("3.0.0"):
```

Comment:
As much as I like the additional safety of parsing, I'm not sure it makes sense to introduce and extra dependency for a single line.
I think we can replace the whole function with `return int(cv2.__version__.split(".")[0])`, that's also what `imutils` is doing, so hopefully it almost always works
https://github.com/jrosebr1/imutils/blob/master/imutils/convenience.py#L216

Reply:
oh yep better indeed!